cluster/src/main/java/com/hartwig/pipeline/CommonArguments.java

```diff
@@ -77,6 +77,8 @@
 
     Optional<String> userLabel();
 
+    Optional<String> panelBedLocation();
```

Comment:
Do you foresee this being used in the batch framework too? Or would it be better in the pipeline-specific arguments?

Reply:
I do foresee. The guys and aus will be doing a bunch of testing with panel